app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
```

Comment:
Rename this to `validate_oidc_request`, to make it consistent with `validate_saml_request`? The OIDC request needs to be validated before `store_request` can be called, so I think this more accurately describes what's happening.